tailwind.config.mjs

```diff
@@ -64,4 +66,4 @@ export default {
       xxl: '1400px',
     },
   },
-} satisfies Config;
+};
```

Comment:
@ahuth I converted this into an `mjs` file so that it can be used without processing in pilots

Reply:
I think it's better to keep TS, so that it's easier for the pilots to write their tailwind files in TS.

Meaning, the pilot swill have to process their tailwind.config.ts anyway, so it's fine for that TS file to import another TS file (the EDS tailwind config).

Reply:
Makes sense to me, reverted to TS